packages/client/wallets/aa/front/services/walletService.ts

```diff
@@ -1,13 +1,13 @@
-import type { SignTypedDataParams } from "@alchemy/aa-core";
```

Comment:
Turns out this is just `Omit<SignTypedDataParameters, "privateKey">`, dropping to remove reference to a dep.